<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bifurcation_widget</class>
 <widget class="QWidget" name="bifurcation_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <widget class="SmoothScrollArea" name="SmoothScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1009</width>
        <height>1011</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="StrongBodyLabel" name="StrongBodyLabel">
         <property name="text">
          <string>Stata variables and parameters shown on axes and their range</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="ComboBox" name="target_variable_x_cbb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Target Variable x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
           <property name="text">
            <string>start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tvx_start_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_3">
           <property name="text">
            <string>end:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tvx_end_edit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="ComboBox" name="target_variable_y_cbb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Target Variable y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_10">
           <property name="text">
            <string>start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tvy_start_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_11">
           <property name="text">
            <string>end:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tvy_end_edit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="ComboBox" name="target_parameter_x_cbb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Target Parameter x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_4">
           <property name="text">
            <string>start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tpx_start_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_5">
           <property name="text">
            <string>end:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tpx_end_edit"/>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_7">
           <property name="text">
            <string>step:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tpx_step_edit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="ComboBox" name="target_parameter_y_cbb">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Target Parameter y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_6">
           <property name="text">
            <string>start:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tpy_start_edit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_8">
           <property name="text">
            <string>end:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tpy_end_edit"/>
         </item>
         <item>
          <widget class="StrongBodyLabel" name="StrongBodyLabel_9">
           <property name="text">
            <string>step:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="tpy_step_edit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="PrimaryPushButton" name="start_btn">
           <property name="text">
            <string>Start Bifurcation Analysis</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>220</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="MplWidget" name="bif_mplWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MplWidget" name="bif_mplWidget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>453</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>zjb.gui.widgets.mpl_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
